Added examples (#58)
LibraryCarpentry/lc-marcedit: _episodes/06-manipulating-MARC-data.md

>  ![Build New Field Window](https://github.com/LibraryCarpentry/lc-marcedit/blob/gh-pages/fig/buildNewField.png)
{: .callout}

> >## Let's build a new 035 9\$a using the data from the 001 and static data

Reviewer: ```suggestion
>## Let's build a new 035 9\$a using the data from the 001
```

>1. Go to Tools → Build New Field
>2. In the new window, in the field box, type in =035  9\$a(LCMarcEdit){001}
>3. Select the box to "Always add this field"

> >3. Click Ok

Reviewer: ```suggestion
>3. Click Process
```

{: .checklist}

>## Use the Build New Field to add a proxy to the 856$u
>

> >1. For eResources, it is sometimes necessary to add your institution's proxy information to the url in the 856$u. Add http:\\exampleproxy.edu/login?url= prefer the url in the 856$u.

Reviewer: ```suggestion
>1. For eResources, it is sometimes necessary to add your institution's proxy information to the url in the 856$u. Add https:\\exampleproxy.edu/login?url= prefix to the url in the 856$u.
```

>
> > ## Solution
> > 1. Go to Tools → Build New Field

> > > 2. In the new window, in the fied box, type in =856  40\$uhttp:\\exampleproxy.edu/login?url={856$u}

Reviewer: ```suggestion
> > 2. In the new window, in the field box, type in =856  40\$uhttps:\\exampleproxy.edu/login?url={856$u}
> > 3. Select the box to "Replace Existing Field"
```

> > > 3. Click Ok

Reviewer: ```suggestion
> > 4. Click Process
```

Reviewer: My MarcEdit (7.5) has Process not Ok

<img src="../fig/selectRecordsForEdit.png" width="650">
<img src="../fig/example_selectRecordsForEdit.png" width="650">

>## Let's add cutters to call numbers in the 099
>1. Go to File → Select Records to Edit
>2. In the new window, type in 099$b in the Field Display

> >3. Click on Import FIle

Reviewer: ```suggestion
>3. Click on Import File
```

> >3. In the search box, type in Display 

Reviewer: ```suggestion
>3. In the search box, type in "Display field not found" to locate all records missing 099$b
```

> >4. Click enter

Reviewer: ```suggestion
>4. Click the magnifying glass icon. A pop up window will indicate the number of records selected that match your criteria. Click OK.
```

> >5. Click on Export Selected Records

Reviewer: ```suggestion
>5. Click on Export Selected. A pop up window indicate that your selected records have been extracted. Click OK.
```

> >6. In the new MarcEditor, go to Tools → Call Number tools → Cuttering Tools → Generate Cutters

Reviewer: ```suggestion
>6. A new MarcEditor window will open. Note the temporary file name at the top of the editor window indicating this file is distinct from your main file. In the new MarcEditor, go to Tools → Call Number tools → Cuttering Tools → Generate Cutters.
> 7. In the field box enter 099 and click Process
> 8. Check your results by using Find All =099
> 9. Select File → Save. A pop up window will indicate your extracted data with edits has been saved back into your main file. Click OK.
```
